src/app/editor/components/container-contents/container-contents.component.ts

```diff
@@ -96,7 +97,12 @@ export class ContainerContentsComponent implements OnInit, OnDestroy {
             .map(this.groupNodesBySchema);
 
         this.schemas$ = this.childrenBySchema$
-            .map(childrenBySchema => Object.values(childrenBySchema).map(nodes => nodes[0].schema));
+            .map(childrenBySchema =>
+                Object.values(childrenBySchema)
+                .map(nodes =>
+                    nodes[0].schema).sort((a: SchemaReferenceFromServer, b: SchemaReferenceFromServer) => {
```

Comment:
put the `.sort(` on the next line below `.map`

Reply:
Also sort first by schema.container (put containers first) and then alphabetically.